.travis.yml

```diff
@@ -51,9 +54,9 @@ linux_setup: &linux_setup
     - "3.6"
     - "3.7"
```

Comment:
Looking at the output of all shards, in each case, exactly 1 python interpreter is used - 3.7.6 for OSXs, 3.6.* for linuxes. As such, a follow up would be good to kill all the extra python setup including lines like `pyenv global 2.7.17 3.6.9 3.7.5 3.8.0` and `./build-support/install_python_for_ci.sh "${PYENV_PY36_VERSION}" "${PYENV_PY37_VERSION}"` which would seem to be either un-needed or doing too much.

Reply:
Negative, we are still testing all the different Python versions within the test itself:

https://github.com/pantsbuild/setup/blob/000271094eea1bc3b00dbde8646a4401e09b6af2/tests/test_sanity_check.py#L44-L50

Reply:
Example of how we are still running Python 2.7: the most recent CI run failed because of Python 2.7 installs. See https://github.com/pantsbuild/setup/pull/65.